src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsStream.cs

```diff
@@ -70,6 +74,7 @@ public TdsStream(TdsWriteStream writeStream, TdsReadStream readStream) : base()
         {
             _writeStream = writeStream;
             _readStream = readStream;
+            _writer = new TdsWriter(this);
```

Comment:
If you decide to make the writer an AutoProperty then you could simply 

```suggestion
            TdsWriter = new TdsWriter(this);
```